ocaml/xapi/xapi_pif.ml

```diff
@@ -459,9 +472,10 @@ let introduce ~__context ~host ~mAC ~device ~managed =
 		"Introducing PIF: device = %s; MAC = %s"
 		device mAC;
 	let mTU = Int64.of_int (Net.Interface.get_mtu dbg ~name:device) in
+	let capabilities = Net.Interface.get_capabilities dbg device in
```

Comment:
Better put this in `introduce_internal`, which is called by `introduce`, `scan`, as well as the special pool-join case.